0_portfolio_input_check_functions.R

```diff
@@ -1092,6 +1092,142 @@ process_raw_portfolio <- function(portfolio_raw,
   return(portfolio_total)
 }
 
+get_fund_coverage <- function(portfolio_raw,
+                              fin_data,
+                              fund_data,
+                              currencies,
+                              grouping_variables) {
+  portfolio <- clean_colnames_portfolio_input_file(portfolio_raw)
+  
+  portfolio <- clear_portfolio_input_blanks(portfolio)
+  
+  portfolio <- add_meta_portfolio(portfolio, inc_meta_portfolio)
```

Comment:
where does `inc_meta_portfolio` come from in this context?

Reply:
it is there... i just copied the process_raw_portfolio function. I think the variable is set FALSE by default in the web_tool_script but may be set TRUE in the current approach of offline testing

Reply:
just as reference: the expectation here is that `inc_meta_portfolio` is in the global environment when this function is called (`inc_meta_portfolio` is one of the variables defined in the yaml parameters file)